EnglishTranslation-master/SKILL.tsv

```diff
@@ -4933,7 +4933,7 @@ SKILL_20150729_004932	Uses the highest abilities between you and allies linked b
 SKILL_20150729_004933	Shrink the size of your target.
 SKILL_20150729_004934	Enlarge the size of your target. Doubles the target's maximum HP, EXP and drop rate.
 SKILL_20150729_004935	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 10 seconds
-SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}{#DD5500}{ol}Lightning{/}{/}Property Additional Damage: +35%{nl}{#DD5500}{ol}Flame{/}{/}Property Target Attack: #{SkillAtkAdd}#
+SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}{#DD5500}{ol}Lightning{/}{/} Property Additional Damage: +35%{nl}{#DD5500}{ol}Flame{/}{/} Property Attack: #{SkillAtkAdd}#
```

Comment:
{#DD5500}{ol}Lightning{/}{/} Property Additional Damage: +35% 
to
Additional {#DD5500}{ol}Lightning{/}{/} Property Damage: +35%

The original does mention 'target' in the last. Does this skill increase your Flame property attack? I don't know how this skill works.